common/middleware/ensure-current-location.js

```diff
@@ -1,14 +1,11 @@
+const { isUndefined } = require('lodash')
+
 function ensureCurrentLocation({ locationsMountpath, whitelist = [] } = {}) {
   return (req, res, next) => {
-    const { permissions = [], locations = [] } = req.session.user || {}
-    const canViewAllMovesWithoutLocations =
-      permissions.includes('moves:view:all') && !locations.length
-
     if (
-      req.session.currentLocation ||
+      !isUndefined(req.session.currentLocation) ||
```

Comment:
this double negative is a bit hard to read could you use: 
```
const currentLocation = get(req, 'session.currentLocation')
 ...
if (
      currentLocation ||
  ...
```

Reply:
Only problem with that is we need it to check for only `undefined` now as when you are viewing all it is set to `null`.